src/store/getters.ts

```diff
@@ -17,7 +17,7 @@ const getters: GetterTree<VQBState, any> = {
   /**
    * current backend's error message
    */
-  backendErrorMessages: (state: VQBState) => state.backendErrors,
+  backendErrorMessages: (state: VQBState) => state.backendMessages,
```

Comment:
I think you can remove this useless getter... In `weaverbird` we allow to get stuff from state directly !